### DISCUSSION TITLE
Set PYTHONDONTWRITEBYTECODE environment variable

### DISCUSSION THREAD

**Discussion by fkromer** [tiangolo/uwsgi-nginx-flask-docker#376] [Questions] [answered]
What do you think about preventing [Python from writing `.pyc` files on the import of source modules](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONDONTWRITEBYTECODE) by adding `ENV PYTHONDONTWRITEBYTECODE 1` to the dockerfiles?

**Comment by tiangolo** [comment, marked as answer]
Yep! I added that a while back to the parent Docker image. :nerd_face:
